scripts/falco-driver-loader

```diff
@@ -535,9 +535,9 @@ load_bpf_probe_compile() {
 	fi
 
 	if [ -n "${BPF_USE_LOCAL_KERNEL_SOURCES}" ]; then
-		local -r kernel_version_major=$(uname -r | cut -d. -f1)
-		local -r kernel_version=$(uname -r | cut -d- -f1)
-		KERNEL_EXTRA_VERSION="-$(uname -r | cut -d- -f2)"
+		local -r kernel_version_major=$(echo "${DRIVER_KERNEL_RELEASE}" | cut -d. -f1)
```

Comment:
Wouldn't it be better to use `KERNEL_RELEASE` here?
I mean, i think we should use `KERNEL_RELEASE` and `KERNEL_VERSION` everywhere, not their `DRIVER_` counterparts.

Reply:
Good catch!

Reply:
The places where I used e.g. `DRIVER_KERNEL_RELEASE` are places where there was an explicit call to `uname -r`. Note that the values of `KERNEL_RELEASE` and `KERNEL_VERSION` change throughout the script, which can be tricky to follow as there is lots of distro-specific logic and special cases.

My idea was to keep existing functionality as is by storing the result of `uname -r` and `uname -v` **once** at the start of the execution, then reuse these values. This allows specifying explicit release and version values as arguments without altering the entire script.

Does that make sense? Did I miss anything?

Reply:
Oh, I see. It makes sense to me now. 

This also adheres to our implicit convention that the `DRIVER_` prefix represents input values (those that may be set using env vars), whereas other vars like `KERNEL_RELEASE` are just for internal use.

:+1: 

Thank you!

N.B.: I've checked the whole PR. Just giving my 2 cents (specifically related to the above comment).

Reply:
Yes, that makes sense to me.